src/components/dex/Trade.vue

```diff
@@ -2,11 +2,12 @@
   <section id="dex--trade">
     <div class="body">
       <div class="side">
-        <div @click="setSide('Buy')" :class="['buy-btn', {selected: side === 'Buy'}]">Buy</div>
-        <div @click="setSide('Sell')" :class="['sell-btn', {selected: side === 'Sell'}]">Sell</div>
+        <button @click="setSide('Buy')" :class="['buy-btn', {selected: side === 'Buy'}]">Buy</button>
```

Comment:
This is for selecting the side right; we have had a lot of complaints about this looking like a button on desktop -- we made it underline the selected one in purple on Desktop recently; can you take a look at what desktop looks like when you get a chance. Doesn't have to be changed in this PR though.